security/cmd/node_agent_k8s/nodeagent.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: extensions/v1beta1
```

Comment:
will be nice to have some high-level description for this config on the top of this file

Reply:
sure. done